<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPreferencesWindow</class>
 <widget class="QDialog" name="ModifyPreferencesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_PPEUsed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PPE Used:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_apply">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>31</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242, 255, 238);</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_segmentInitialCycle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Segment the initial cycle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_notification">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Applied changes will take effect immediately.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_defaultToWorkingDirectory">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242, 255, 238);</string>
   </property>
   <property name="text">
    <string>Set Default to Current Working Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_defaultEditor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Default Editor:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_close">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>31</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242, 255, 238);</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_defaultLocationsFile">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_defaultDatabaseName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Default Database Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_defaultLocationsFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Default Locations File:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_defaultWorkingDirectory">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_defaultParametersFile">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_defaultWorkingDirectory">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Default Working Directory:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_defaultDatabaseName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_defaultVariant">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_defaultEditor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_plotsAsStands">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Process plots as stands:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_defaultParametersFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Default Parameters File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_defaultVariant">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Default Variant:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_defaultUsePPE">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_defaultSegmentCycle">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_defaultProcessPlots">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>button_close</tabstop>
  <tabstop>lineEdit_defaultVariant</tabstop>
  <tabstop>lineEdit_defaultLocationsFile</tabstop>
  <tabstop>lineEdit_defaultParametersFile</tabstop>
  <tabstop>lineEdit_defaultDatabaseName</tabstop>
  <tabstop>button_defaultUsePPE</tabstop>
  <tabstop>lineEdit_defaultEditor</tabstop>
  <tabstop>button_defaultSegmentCycle</tabstop>
  <tabstop>button_defaultProcessPlots</tabstop>
  <tabstop>lineEdit_defaultWorkingDirectory</tabstop>
  <tabstop>button_apply</tabstop>
  <tabstop>button_defaultToWorkingDirectory</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
